### ISSUE TITLE
How to generate the score for a determined region of an image using Mask R-CNN

### ISSUE THREAD

**Issue by mingqiJ** [pytorch/vision#7043]
### 🐛 Describe the bug

I want to change the RegionProposalNetwork of Mask R-CNN to generate the score for a determined region of an image using Mask R-CNN.

```
import torch
from torch import nn
import torchvision.models as models
import torchvision
from torchvision.models.detection import MaskRCNN
from torchvision.models.detection.anchor_utils import AnchorGenerator

model = models.detection.maskrcnn_resnet50_fpn(pretrained=True)

class rpn_help(nn.Module):
    def __init__(self,) -> None:
        super().__init__()
    def forward(self,) :
        proposals=torch.tensor([ 78.0753,  12.7310, 165.6465, 153.7253])
        proposal_losses=0
        return proposals, proposal_losses

model.rpn= rpn_help
model.eval()
model(input_tensor) # input_tensor is an image
```

It takes error like this
<img width="786" alt="WeChatbb686829d6c0f06106e53c1e3feecb55" src="https://user-images.githubusercontent.com/98499594/208374946-137eb9b2-6a64-4a06-8d4e-57caf1bb72b3.png">

Does anyone know how to generate the score for a determined region of an image using Mask R-CNN
?

### Versions

python 3.8
